frontend/public/components/utils/pdb.tsx

```diff
@@ -0,0 +1,8 @@
+import { LabelSelector } from '@console/internal/module/k8s';
+
+export const getPDBResource = (pdbResources, resource) =>
```

Comment:
Type please. Also `resource` should have a generic type that is not `K8sResourceKind` as we are expecting a path here in `spec`.